### PR TITLE
Newsweek Clone page with Bootstrap

### PR DESCRIPTION
**Pull request by queeksm**
This is a clone of the newsweek main page made using HTML/CSS and Bootstrap

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,2041 @@
...
+        <div class="col-6 d-flex justify-content-center">
+          <a href="#">
+            <img src="img/Newsweek.png" alt="logo">
+          </a>
+        </div>
+        <div class="col align-self-end lchild">
+          <a href="#" class="align-self-center white-bold "> SIGN IN</a>
+          <button type="button" class="btn btn-white redText ">
+            SUBSCRIBE >
+          </button>
```
Comment (pelzolga123) [inline index.html:35]: `Subscribe` should be a link.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Requires changes.

**Comment by pelzolga123** [comment]
## Great  work :+1:
### Your project looks amazing! But more changes are required:
* [ ] Please check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). Now it shows  error.

* [ ] Subscribe should be a link like on original website.

* [ ] This section in Tablet version needs a little correction:
image
Original:
image

* [ ] Mobile version needs to be more responsive
image
Original:
image

* [ ] `Latest news` section needs to be responsive:
image

Original:
image

* [ ] Footer needs some adjustment:
image

* [ ] You have a lot of css in this project, but main goal is to use as many bootstrap classes as possible. Please try to reduce amount of your css code and add more bootstrap. [Link to screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/using_bootstrap/bootstrap_classes_requirement.png)

### Please kindly Fix the above issue and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Review by nikasulo — APPROVED** [review]
# This is so great 👏
Please merge and see you at the top!  :rocket:

**Pull request merged**